Address/src/Http/Response.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+namespace Address\Http {
+
+    use Address\Entities\Address;
+    use Address\Entities\Text;
+
+    class Response
+    {
+        /** @var string */
+        private $redirect;
+
+        /** @var array */
+        private $addresses;
+
+        /** @var Address */
+        private $address;
+
+        /** @var array */
+        private $texts;
+
+        /** @var Text */
+        private $text;
+
+        public function setRedirect(string $path)
+        {
+            $this->redirect = $path;
+        }
+
+        public function getRedirect(): string
+        {
+            return $this->redirect;
+        }
+
+        public function hasRedirect(): bool
+        {
+            return isset($this->redirect);
+        }
+
+        public function setAddress(Address $address)
+        {
+            $this->address = $address;
+        }
+
+        public function getAddress(): Address
+        {
+            return $this->address;
+        }
+
+        public function setAddresses(array $addresses)
```

Comment:
http://stackoverflow.com/questions/34273367/type-hinting-in-php-7-array-of-objects